src/lib/getData/getSubpage.ts

```diff
@@ -23,6 +23,28 @@ export const GET_SUBPAGE = gql`
       locale
       page {
         slug
+        subpages(sort: "start:desc", limit: 5) {
```

Comment:
It shouldn't be limited as we will show all the subpages in the main page.
We may show in the sidebar only first five items. However it's better to limit popular items which correspond most-recent ones.